### ISSUE TITLE
Skipping docs deployment - GITHUB_API_KEY not found

### ISSUE THREAD

**Issue by jpwspicer** [JuliaDocs/Documenter.jl#353]
Hi, I am using Julia 0.5.0 and Documenter v0.7.1. 

For some reason, my (unregistered) package [Vallado.jl](https://github.com/jpwspicer/Vallado.jl) is skipping docs deployment at very build. After reading [this](https://github.com/JuliaDocs/Documenter.jl/issues/81) issue I added a `DOCUMENTER_DEBUG=true` key to travis, giving this output:

```
Documenter: populating indices.
Documenter: rendering document.
Documenter: copying assets to build directory.
 ?? TRAVIS_REPO_SLUG       = "jpwspicer/Vallado.jl"
 ?? TRAVIS_PULL_REQUEST    = "false"
 ?? TRAVIS_OS_NAME         = "linux"
 ?? TRAVIS_JULIA_VERSION   = "release"
 ?? TRAVIS_BRANCH          = "gh-pages"
 ?? TRAVIS_TAG             = ""
 ?? git commit SHA         = f2bfd80
 ?? GITHUB_API_KEY exists  = false
 ?? DOCUMENTER_KEY exists  = true
 ?? .documenter.enc path   = /home/travis/.julia/v0.5/Vallado/docs/.documenter.enc
 ?? .documenter.enc exists = false
 ?? should_deploy          = false
Documenter: skipping docs deployment.
```

This implies that the Github public key cannot be found, but I'm pretty sure I've set both keys up correctly according to the instructions. Github displays "Never used" next to the documenter key, so it's not being found. 

Docs deployment is also skipped when I `cd` into `docs` and run `julia make.jl`, but I can't view the log files in this case.

Any advice would be appreciated, thanks!

**Comment by MichaelHatherly** [comment]
The version of Julia named in `docs/make.jl` must match one of those named in `.travis.yml`, i.e. https://github.com/jpwspicer/Vallado.jl/blob/00f2165c614e44806d029d8f5d4b80d2731cee3a/.travis.yml#L9 and https://github.com/jpwspicer/Vallado.jl/blob/00f2165c614e44806d029d8f5d4b80d2731cee3a/docs/make.jl#L8 must either both be `release`, or `0.5.0`.

**Comment by jpwspicer** [comment]
Many thanks for the quick response! I have changed both to `release`.

- On Github, branch `gh-pages` has now been built by `autodocs` and builds in travis - seems good. 
- However `master` branch now fails in travis - will troubleshoot. 
- Vallado's ReadTheDocs page does not seem to have changed.

**Comment by MichaelHatherly** [comment]
> However master branch now fails in travis - will troubleshoot. 

That will be #351, which will be fixed later today as part of #352.

> Vallado's ReadTheDocs page does not seem to have changed. 

Documenter doesn't use ReadTheDocs, the docs will be hosted on https://jpwspicer.github.io/Vallado.jl/latest/.

**Comment by jpwspicer** [comment]
Thanks very much for your help! The documenter side of things seems to work now, so closing this out.
